### ISSUE TITLE
Talk is cheap,show me code.

### ISSUE THREAD

**Issue by jixunx** [borisbanushev/stockpredictionai#134]

**Comment by simonblack** [comment]
I don't think anyone got a copy

**Comment by vesl** [comment]
Yes Arima cant predict stock price so precisely, on top of that, if arima predict price so well, thats enough, you rich. 
No need to add NLP or GAN or something else.
That's fake obviously dont waste your time guys.

**Comment by IISuperluminaLII** [comment]
idk man mine was giving pretty good predictions (~40% accurate future prediction, good enough indicator for me to algo trade) , also just shows the laziness of all the "Code requestors" I literally read this and could understand about all of the programming talk. You guys need to just read it and implement it along the way rather than wanting it spoon fed ffs

**Comment by vesl** [comment]
> idk man mine was giving pretty good predictions (~40% accurate future prediction, good enough indicator for me to algo trade) , also just shows the laziness of all the "Code requestors" I literally read this and could understand about all of the programming talk. You guys need to just read it and implement it along the way rather than wanting it spoon fed ffs

Man please read his paper again ... 
That his ARIMA predictions https://github.com/borisbanushev/stockpredictionai/raw/master/output_54_0.png , do you really think that's not a fake ?
